app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -159,4 +164,22 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
         EitherT.pure[Future, String, UserInfo](UserInfo.AlreadyHasAccount(value))
 
     }
+
+  def createAccount: Action[AnyContent] = authorisedFromStride { implicit request ⇒
+    checkSession(SeeOther(routes.StrideController.getEligibilityPage().url),
+                 checkIsEligible(result ⇒ helpToSaveConnector.createAccount(result.nSIUserInfo)
+        .fold(
+          error ⇒ {
+            logger.warn(s"error during create account call, error: $error")
+            InternalServerError(views.html.technical_error())
```

Comment:
you need to call this `internalServerError()` as shown in the other endpoints in this controller

Reply:
ok technical_error page removed, thanks!